vsintegration/tests/UnitTests/SemanticColorizationServiceTests.fs

```diff
@@ -33,15 +33,15 @@ type SemanticClassificationServiceTests() =
     let getRanges (sourceText: string) : (Range.range * SemanticClassificationType) list =
```

Comment:
Nit: can we rename `sourceText` to `source` or `sourceFile` so that the subsequent call is `SourceText.From(sourceFile)` instead of `SourceText.From(sourceText)`?